source/controller/project_controller.py

```diff
@@ -299,7 +299,7 @@ def on_check_condition(self):
 
     def on_invalid_graph_display_alert(self):
         index = self.project_tool_bar.combo_graphs.currentIndex()
-        self.project_tool_bar.combo_graphs.setItemText(index, f'Graph {index} - Invalid')
+        self.project_tool_bar.combo_graphs.setItemText(index, f'Graph {index+1} - Invalid')
```

Comment:
@fsoupimenta  I found the problem, when the invalid graph window was called, the combobox counter was reogarnized, but the index received was from 0 to n-1 unlike how our combobox works.